docs/install/ubuntu_setup.md

```diff
@@ -140,9 +140,30 @@ On Ubuntu versions 16.04 or later, you need the following dependencies:
 **Step 1:** Install build tools and git.
 ```bash
     sudo apt-get update
-    sudo apt-get install -y build-essential git
+    sudo apt-get install -y build-essential git ninja-build ccache
 ```
 
+**For Ubuntu 18.04 and CUDA builds you need to update CMake**
```

Comment:
What is the implication for older Ubuntu OS? Is there a version >= that must be met?

Reply:
For older versions the distribution's provided CMake is working.

Reply:
Should we clarify that in the doc?

Reply:
No, I guess it is fine if there is no action required.